samples/topology/src/main/java/quarks/samples/topology/CombiningStreamsProcessingResults.java

```diff
@@ -0,0 +1,158 @@
+/*
```

Comment:
Should the IBM copyright be removed in favor of the ASF license header? Or should the license be added in addition to the copyright?